.github/workflows/stale.yml

```diff
@@ -0,0 +1,29 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  invalid-to-stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          stale-issue-message: 'This issue is invalid or waiting response because it has been open 7 days with no activity. Remove stale label or comment or this will be closed in 5 days.'
+          days-before-issue-stale: 7
+          only-labels: 'invalid,awaiting response'
+
+  stale-close:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in 5 days.'
```

Comment:
This issue is marked as stale because there has been no activity in the past 30 days. It will be closed in 5 days if the stale label is not removed or if there is no further response.